src/Diagnostics.DataProviders/KustoClient.cs

```diff
@@ -40,9 +40,10 @@ private HttpClient _httpClient
             }
         }
 
-        public KustoClient(string requestId)
+        public KustoClient(KustoDataProviderConfiguration config, string requestId)
         {
             _requestId = requestId;
+            KustoApiQueryEndpoint = config.KustoApiEndpoint + ":443/v1/rest/query";
```

Comment:
Since config has kusto api end point now, does this require a change in appsettings.json?

Reply:
No as this is generated from the `AADKustoResource` which is already in the app.settings. See [Configuration](https://github.com/Azure/Azure-AppServices-Diagnostics/pull/163/files#diff-4bf3b7af0397fb900e551a8192d4b5ae)

Reply:
I see. I understood now. Thanks Hawk